operator/operator_server.go

```diff
@@ -0,0 +1,64 @@
+package operator
+
+// since operator is a serverless component on its own, its associated server
+// is only used for hosting the /metrics, /livez and /readyz endpoints
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type OperatorServer struct {
+	Mux  *http.ServeMux
```

Comment:
Tried keeping the Mux private as well as OperatorServer private to the go package, but that didn't fly so well when using it with the now deprecated `simple/operator.go`.

Reply:
Could go back to private if we can find another way to keep the deprecated code building.